Resources/UI/Camera/README.md

```diff
@@ -0,0 +1,78 @@
+# Camera
+
+A UI component that simplifies handling with camera. `PhotoCamera` & `QRCodeScanner`.
+
+### PhotoCamera Example:
+```swift
+let camera = PhotoCamera()
+func prepareCamera() {
+  camera.prepare()
```

Comment:
What's the purpose of `prepare` method we need to trigger manually? Should it be implicitly executed instead?

Reply:
Will change `prepare` to an async method so the user has feedback if something failed during configuration.